cli/dts/lib.deno.ns.d.ts

```diff
@@ -163,6 +163,27 @@ declare namespace Deno {
    * */
   export function test(name: string, fn: () => void | Promise<void>): void;
 
+  /** Register a test which will be run when `deno test` is used on the command
+   * line and the containing module looks like a test module.
+   * `fn` can be async if required.
+   *
+   * ```ts
+   * import {assert, fail, assertEquals} from "https://deno.land/std/testing/asserts.ts";
+   * 
+   * Deno.test("My test description", ():void => {
+   *   assertEquals("hello", "hello");
+   * }, { only: true });
+   * 
+   * Deno.test("My async test description", async ():Promise<void> => {
+   *   const decoder = new TextDecoder("utf-8");
+   *   const data = await Deno.readFile("hello_world.txt");
+   *   assertEquals(decoder.decode(data), "Hello world");
+   * }, { only: true });
+   * ```
+   */
+  export function test(name: string, fn: () => void | Promise<void>, options?: TestOptions): void;
+  type TestOptions = Omit<TestDefinition, "name" | "fn">;
```

Comment:
I think you should export this type, @nayeemrmn what do you think?

Reply:
I said what I think here: https://github.com/denoland/deno/pull/7513#discussion_r489363224.